src/uct/tcp/tcp_iface.c

```diff
@@ -128,6 +128,7 @@ static ucs_status_t uct_tcp_iface_get_device_address(uct_iface_h tl_iface,
 
     if (ucs_sockaddr_is_inaddr_loopback(saddr)) {
         dev_addr->flags |= UCT_TCP_DEVICE_ADDR_FLAG_LOOPBACK;
+        memset(pack_ptr, 0, sizeof(uct_iface_local_addr_ns_t));
```

Comment:
is it some bug fix?

Reply:
yes, it's an uninitialized memory issue found with valgrind, probably because we are calling get_device_address from another flow with this PR